### PR TITLE
U1-88 Add tests for cli option validators

### PR DESCRIPTION
**Pull request by duyphaphach**
JIRA Ticket: https://pass-career.atlassian.net/browse/U1-88

### CODE REVIEW

File: source/helpers/orientation-validator.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-else-return */
+import {ERROR_CODES, ORIENTATION_ALL, ORIENTATION_LANDSCAPE, ORIENTATION_PORTRAIT} from '../constants';
```
Comment (knightus) [inline source/helpers/orientation-validator.js]: Codacy Issue found: [Expected a line break before this closing brace.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5038858)

**Comment by knightus** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5038858&bid=16595404)

File: source/helpers/path-validator.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-else-return */
+import {ERROR_CODES} from '../constants';
```
Comment (knightus) [inline source/helpers/path-validator.js]: Codacy Issue found: [A space is required before '}'.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5038858)

File: source/helpers/orientation-validator.js (hunk 1 above)
Comment (knightus) [inline source/helpers/orientation-validator.js]: Codacy Issue found: [A space is required before '}'.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5038858)

File: tests/unit/helpers/name-pattern-validator.test.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import namePatternValidator from '../../../source/helpers/name-pattern-validator';
+import {IMAGE_NAME_FORMAT_HASH, IMAGE_NAME_FORMAT_DATE, IMAGE_NAME_FORMAT_ORIGIN} from '../../../source/constants';
```
Comment (knightus) [inline tests/unit/helpers/name-pattern-validator.test.js]: Codacy Issue found: [A space is required before '}'.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5038858)

File: source/index.js [hunk 4]
```diff
@@ -6,7 +6,7 @@
~  IMAGE_NAME_FORMAT_DATE,
~  ORIENTATION_LANDSCAPE,
~  ORIENTATION_PORTRAIT,
~  ORIENTATION_ALL, ERROR_CODES,
~} from './constants';
~import {
~  getImages,
```
Comment (knightus) [inline source/index.js]: Codacy Issue found: [Missing trailing comma.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5038858)

File: tests/unit/helpers/orientation-validator.test.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import orientationValidator from '../../../source/helpers/orientation-validator';
+import {ORIENTATION_ALL, ORIENTATION_LANDSCAPE, ORIENTATION_PORTRAIT} from '../../../source/constants';
```
Comment (knightus) [inline tests/unit/helpers/orientation-validator.test.js]: Codacy Issue found: [A space is required before '}'.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5038858)

**Review by huyhqit — APPROVED** [review]

**Pull request merged**